py-polars/polars/_utils/construction/series.py

```diff
@@ -510,15 +511,17 @@ def numpy_to_pyseries(
             strict=strict,
             nan_to_null=nan_to_null,
         )
-        return (
```

Comment:
We definitely don't want to go through the engine during construction. So added `reshape` directly on `Series`.